src/components/NotificationsSystem.js

```diff
@@ -63,8 +67,8 @@ export class NotificationsSystem extends Component {
     if (windowWidth < theme.smallScreenMin) {
       return (
         <NotificationsContainer
-          key='t'
-          position='t'
+          key={theme.smallScreenPosition || POSITIONS.top}
```

Comment:
use a variable for `theme.smallScreenPosition || POSITIONS.top` to avoid repetition